### PR TITLE
Add documentation for Python Script Engine

### PR DESCRIPTION
**Pull request by sophiesongge**

### CODE REVIEW

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -161,6 +162,14 @@ relies on script task and is a nice introduction to workflows.
 Scripts can also be used to decorate tasks with specific actions, we support
 <<_pre_post_clean,pre>>, <<_pre_post_clean,post>>, <<_pre_post_clean,clean>> and <<_selection,selection>> scripts.
 
+==== Python Script Engine
+
+Since there are many different versions of Python (mainly Python2 and Python3) which are not compatible, ProActive supports all the versions of python.
```
Comment (marcocast) [inline src/docs/user/ProActiveUserGuide.adoc]: I think it is more clear to say : 
"ProActive supports all the Python versions (Python2, Python3,etc)"

File: src/docs/user/ProActiveUserGuide.adoc [hunk 2]
```diff
@@ -161,6 +162,14 @@ relies on script task and is a nice introduction to workflows.
...
+By default, the Python used to execute the Script is the default Python version on your machine. In order to use another Python version to execute
+the task, it is required to put a key 'PYTHON_COMMAND' in Generic Info, and its value is the path (symbolic or absolute, for example 'python3' or
```
Comment (marcocast) [inline src/docs/user/ProActiveUserGuide.adoc]: better to specify Generic Info at task or job level?

File: src/docs/user/ProActiveUserGuide.adoc (hunk 1 above)
Comment (sophiesongge) [reply src/docs/user/ProActiveUserGuide.adoc]: Done!

File: src/docs/user/ProActiveUserGuide.adoc (hunk 2 above)
Comment (sophiesongge) [reply src/docs/user/ProActiveUserGuide.adoc]: Done! Good find!

**Pull request merged**